src/c_api/c_api.cc

```diff
@@ -1363,7 +1363,12 @@ int MXGetVersion(int *out) {
 #if MXNET_USE_TVM_OP
 int MXLoadTVMOp(const char *libpath) {
   API_BEGIN();
-  tvm::runtime::TVMOpModule::Get()->Load(libpath);
+  tvm::runtime::TVMOpModule *libpath_module =  tvm::runtime::TVMOpModule::Get();
+  libpath_module->Load(libpath);
+  tvm::runtime::TVMOpModule *cubin_module =  tvm::runtime::TVMOpModule::Get();
```

Comment:
tvm::runtime::TVMOpModule::Get() returns a singleton, thus `libpath_module` and `cubin_module` are essentially the same object.